lib/ansible/cli/doc.py

```diff
@@ -312,10 +312,10 @@ class DocCLI(CLI, RoleMixin):
     _RULER = re.compile(r"\bHORIZONTALLINE\b")
 
     # rst specific
-    _REFTAG = re.compile(r":ref:")
-    _TERM = re.compile(r":term:")
-    _NOTES = re.compile(r".. note:")
-    _SEEALSO = re.compile(r"^\s*.. seealso:.*$", re.MULTILINE)
+    _RST_NOTE = re.compile(r".. note::")
```

Comment:
Looking at how this is handled, I wonder if this could be turned into a sequence of transformations.
```python
    _RST_TRANSFORMATIONS = (
        (re.compile(r".. seealso::"), "See website for:"),
        (re.compile(r".. note::"), "Note:"),
        (re.compile(r":\w+?:`"), "website for `"),
        (re.compile(r".. \w+?::"), ""),
    )
```

And then just
```python
for regex, replacement in cls._RST_TRANSFORMATIONS:
    t = regex.sub(replacement, t)
```
(below)